controllers/controllers.go

```diff
@@ -88,3 +90,36 @@ func checkCSRF(r *http.Request, token string) bool {
 	}
 	return true
 }
+
+func BadRequest(w http.ResponseWriter, r *http.Request) {
```

Comment:
__[golint]__
* [golint] exported function BadRequest should have comment or be unexported